### PR TITLE
Update `slides_create_image` comment.

### PR DESCRIPTION
**Pull request by grant**
Standardizes the `slides_create_image` comment.

### CODE REVIEW

File: slides/snippets/snippets.js [hunk 1]
```diff
@@ -122,12 +122,6 @@ function createTextboxWithText(presentationId, pageId, callback) {
 function createImage(presentationId, pageId, imageFilePath, imageMimetype, callback) {
```
Comment (erickoledadevrel) [inline slides/snippets/snippets.js]: As per other samples, remove file path and mime type parameters.

Comment (grant) [reply slides/snippets/snippets.js]: Done.

Comment (grant) [reply slides/snippets/snippets.js]: Done.

**Comment by grant** [comment]
Should be good to review now.

**Pull request merged**